develop/tutorials/articles/110-portlets/07-using-javascript-in-your-portlets/03-using-npm-in-your-portlets/02-configuring-liferay-npm-bundler.markdown

```diff
@@ -105,6 +132,17 @@ only applied to packages with this name and *any* version.
 `some-package-name@version` scope is only applied to packages with the specified 
 *package name* and *version*.
 
++$$$
+
+**Note:** Prior to version 1.4.0 of the liferay-npm-bundler, package 
+configurations were placed next to the tools options 
+(`*`, `output`, `exclude`, etc.) To prevent package name collisions, package 
+configurations are now namespaced and placed under the `packages` section. To 
+maintain backwards compatibility, the liferay-npm-bundler falls back to the root 
+section outside `packages` for package configuration.
+
+$$$
```

Comment:
@izaera By
> the liferay-npm-bundler falls back to the root section outside `packages` for package configuration
Do you mean the bundler reads the package configuration found under `packages` and also looks for packages at the root section for backwards compatibility.

Reply:
Yes, but it only looks outside `packages` if no match is found inside `packages`. Note that a match can be package+version, only package, or *.

Reply:
Ok. Thanks!